### PR TITLE
Issue #6367 fixed

### PR DESCRIPTION
**Pull request by suraj-webkul**
Issue #6367
The percentage of the discount now visible on the product view page if there is any discount for the product.

### CODE REVIEW

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss [hunk 1]
```diff
@@ -56,6 +56,8 @@ body {
 
 .card-discount {
     color: rgba(38, 163, 124, 0.83);
+    padding-left: 5px;
+    font-size: 15px !important
```
Comment (akathuria289) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss:60]: @suraj-webkul remove important from font-size

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Need to **improve the UI**. please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63f6ff4793ea2803a945fb35)

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Any update?

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss]: Removed

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Still getting the spacing issue. there should be a space of 4px between all prices.

**Comment by suraj-webkul** [comment]
Please publish assests all changes are done

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not fix** for **bundle** product, please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63fc52a2aac9534fac273b8b)

**Comment by Sanjaybhattwebkul** [comment]
@devansh-webkul 
Please review this pr.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by akathuria289 — APPROVED** [review]
@suraj-webkul resolve the conflict

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
resolve the duplicate commits.

File: packages/Webkul/Product/src/Type/Bundle.php [hunk 2]
```diff
@@ -410,7 +416,7 @@ public function validateBundleOptionForCart($data)
         foreach ($data as $key => $value) {
             if (is_array($value)) {
                 $data[$key] = $this->validateBundleOptionForCart($value);
-            } elseif ($value) {
+            } elseif ($value && $value) {
```
Comment (akathuria289) [inline packages/Webkul/Product/src/Type/Bundle.php:419]: remove one $value

**Review by akathuria289 — DISMISSED** [review]

**Comment by Amitk-webkul** [comment]
@suraj-webkul 
Not fixed for bundle product, please check the reference [Here](https://webkul.chatwhizz.com/share/screenshot/642aa98a2a666707c4096f04)

File: packages/Webkul/Product/src/Type/Bundle.php (hunk 2 above)
Comment (suraj-webkul) [reply packages/Webkul/Product/src/Type/Bundle.php:419]: done

**Comment by suraj-webkul** [comment]
> @suraj-webkul Not fixed for bundle product, please check the reference [Here](https://webkul.chatwhizz.com/share/screenshot/642aa98a2a666707c4096f04)

Please try to run 
~~~
npm run watch  
~~~

After testing please run
~~~
npm run dev
~~~

in Velocity package/module

**Comment by Amitk-webkul** [comment]
@suraj-webkul 
Please resolve the conflicts.

**Pull request closed without merging**